src/main/kotlin/blackjack/domain/Players.kt

```diff
@@ -0,0 +1,32 @@
+package blackjack.domain
+
+class Players private constructor(
+    val members: List<Player>,
+) {
+    val playerIterator: Iterator<Player> = members.iterator()
+
+    fun drawDefaultCards(
+        deck: Deck,
+        cardCount: Int,
+    ) {
+        members.forEach { player -> repeat(cardCount) { player.addCard(deck.draw()) } }
+    }
+
+    fun getNextPlayer(): Player {
+        return playerIterator.next()
+    }
+
+    fun hasBlackJackPlayer(): Boolean {
+        return members.any { it.isBlackJack() }
+    }
+
+    fun hasHitPlayer(): Boolean {
+        return members.any { it.status == PlayerStatus.HIT }
+    }
+
+    companion object {
+        fun from(names: List<String>): Players {
+            return Players(names.map { Player(it) })
```

Comment:
요기도 함수 reference를 활용해볼 수 있겠네요!
```suggestion
            return Players(names.map(::Player))
```

Reply:
이부분은 Player 클래스가 2개의 프로퍼티를 받도록 되어 있는데 default parameter를 설정해줘도 프로퍼티가 2개면 메서드참조가 안되더라구요!